### DISCUSSION TITLE
Import LiquidOptions type directly

### DISCUSSION THREAD

**Discussion by Daniel-Knights** [harttle/liquidjs#590] [Q&A] [answered]
Hi, I need to use the `LiquidOptions` type in my Typescript project, but can't seem to access it directly. Is there a way to do this?

My current workarounds are to either import directly from `liquidjs/dist/liquid-options.js` or use the `ConstructorParameters` utility type to extract it from the `Liquid` class.

**Comment by harttle** [comment, marked as answer]
I believe it's the best way to access its type for now. I can expose `LiquidOptions` in next minor version, created #593 to track.

**Reply by harttle** [reply]
update: please try 10.6.2
